From 0c4b65b6660ad646eba31e715bb72301a89e61ed Mon Sep 17 00:00:00 2001
From: Yuan Liao <liaoyuan@gmail.com>
Date: Mon, 26 Jul 2021 20:38:02 -0700
Subject: [PATCH] Skip tests that require Internet connectivity

Some tests may attempt to connect to a URL, but such access will be
denied when FEATURES="network-sandbox" is enabled, causing test failure.

Signed-off-by: Yuan Liao <liaoyuan@gmail.com>
---
 libraries/stdlib/jvm/test/io/ReadWrite.kt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libraries/stdlib/jvm/test/io/ReadWrite.kt b/libraries/stdlib/jvm/test/io/ReadWrite.kt
index fc57ed1fb496..5d2941777164 100644
--- a/libraries/stdlib/jvm/test/io/ReadWrite.kt
+++ b/libraries/stdlib/jvm/test/io/ReadWrite.kt
@@ -124,6 +124,7 @@ class ReadWriteTest {
         file.deleteOnExit()
     }
 
+    @Ignore
     @Test fun testURL() {
         val url = URL("http://kotlinlang.org")
         val text = url.readText()
-- 
2.31.1

